lib/views/widgets/pages_widget.dart

```diff
@@ -1,11 +1,11 @@
 import 'package:cycle_lock/bindings/auth_binding.dart';
```

Comment:
This file does not have widgets. Move the `getPages` function to a `navigation` folder?

Reply:
yeah Will migrate those objects, navigation objects inside navigation folder and constant objects to constant folder